### PR TITLE
Hook into Pods 2.7.26 shortcode current post detection filter

### PR DESCRIPTION
**Pull request by sc0ttkclark**
This hooks into the new Pods 2.7.26 filter added to shortcodes to detect current post (default: only if `in_the_loop()`)

### CODE REVIEW

File: pods-beaver-themer.php [hunk 1]
```diff
@@ -110,25 +109,23 @@ function pods_beaver_enqueue_assets() {
...
  *
  * @since 1.3.3
  */
 function pods_beaver_fake_loop_start() {
-
-	add_action( 'fl_builder_loop_before_query', 'pods_beaver_fake_loop_true');
-
+	add_filter( 'pods_shortcode_detect_from_current_post', '__return_true', 9 );
+	//properly clean up after the loop
+	add_action( 'loop_end', 'pods_beaver_fake_loop_end');
```
Comment (sc0ttkclark) [inline pods-beaver-themer.php:119]:
```suggestion
	add_filter( 'pods_shortcode_detect_from_current_post', '__return_true', 9 );

	// Remove the hook after the end of the loop.
	add_action( 'loop_end', 'pods_beaver_fake_loop_end' );
```

**Review by quasel — DISMISSED** [review]
Tested, should work properly now!

**Review by quasel — APPROVED** [review]

**Review by quasel — DISMISSED** [review]

**Review by nicdford — APPROVED** [review]
This looks great! Awesome work @quasel!!! 🎉

**Comment by nicdford** [comment]
and @sc0ttkclark !! :)

File: pods-beaver-themer.php [hunk 2]
```diff
@@ -82,10 +81,10 @@ function pods_beaver_init() {
  */
 function pods_beaver_admin_nag() {
 
-	if ( is_admin() && ( ! class_exists( 'FLBuilder' ) || ! defined( 'PODS_VERSION' ) ) ) {
+	if ( is_admin() && ( ! class_exists( 'FLBuilder' ) || ! defined( 'PODS_VERSION' ) || version_compare( PODS_VERSION, '2.7.26', '<' ) ) ) {
```
Comment (sc0ttkclark) [inline pods-beaver-themer.php:84]:
```suggestion
	if ( ! class_exists( 'FLBuilder' ) || ! defined( 'PODS_VERSION' ) || version_compare( PODS_VERSION, '2.7.26', '<' ) ) {
```

**Review by quasel — DISMISSED** [review]

**Review by sc0ttkclark — COMMENTED** [review]
Passes known QA cases

**Comment by sc0ttkclark** [comment]
Holding on merge/release for Pods 2.7.26 release

**Pull request merged**